app/assets/stylesheets/modules/top_navbar.scss

```diff
@@ -38,8 +37,36 @@
  }
 }
 
+.navbar {
+  max-height: 120px;
+
+  .header-bar {
+    flex-wrap: wrap !important;
+  }
+
+  #topnav-container {
+    padding: 2px 0;
+    height: 29px;
+  }
+}
+
 @include media-breakpoint-up(xs) {
   #topnav {
     width: 100%;
   }
 }
+
+@media (min-width: 768px) {
```

Comment:
Can we use the bootstrap defined breakpoints here? https://getbootstrap.com/docs/4.3/layout/grid/#example-usage

Reply:
It does not appear to be needed at all right now, so I am just removing it. I will dig into the using bootstrap breakpoints later if needed...